rktlet/runtime/helpers.go

```diff
@@ -313,6 +313,20 @@ func generateAppSandboxCommand(req *runtimeApi.RunPodSandboxRequest, uuidfile st
 		// https://github.com/kubernetes/kubernetes/pull/33709 is merged.
 	}
 
+	// Add port mappings only if it's not hostnetwork.
+	linuxNamespaces := getLinuxNamespaces(req)
```

Comment:
The autogenerated getters are "nil safe", so you shouldn't have to nilcheck if you do e.g. `req.GetConfig().GetLinux().GetNamespaceOptions()`

For what I mean by nil-safe, they're all generated like [this](https://github.com/kubernetes/kubernetes/blob/4adb23c8ccedd1819fc9396c75a28cf7f552bb59/pkg/kubelet/api/v1alpha1/runtime/api.pb.go#L1496-L1501)

Given that, I think we can drop the helper function.